Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -12,233 +11,234 @@
 using System.Threading.Tasks;
 using System.Threading;
 
-namespace Flow.Launcher.Plugin.BrowserBookmark
+namespace Flow.Launcher.Plugin.BrowserBookmark;
+
+public class Main : ISettingProvider, IPlugin, IReloadable, IPluginI18n, IContextMenu, IDisposable
 {
-    public class Main : ISettingProvider, IPlugin, IReloadable, IPluginI18n, IContextMenu, IDisposable
-    {
-        private static PluginInitContext context;
+    private static PluginInitContext _context;
 
-        private static List<Bookmark> cachedBookmarks = new List<Bookmark>();
+    private static List<Bookmark> _cachedBookmarks = new List<Bookmark>();
 
-        private static Settings _settings;
+    private static Settings _settings;
 
-        private static bool initialized = false;
+    private static bool _initialized = false;
 
-        public void Init(PluginInitContext context)
-        {
-            Main.context = context;
+    public void Init(PluginInitContext context)
+    {
+        _context = context;
 
-            _settings = context.API.LoadSettingJsonStorage<Settings>();
+        _settings = context.API.LoadSettingJsonStorage<Settings>();
 
-            LoadBookmarksIfEnabled();
-        }
+        LoadBookmarksIfEnabled();
+    }
 
-        private static void LoadBookmarksIfEnabled()
+    private static void LoadBookmarksIfEnabled()
+    {
+        if (_context.CurrentPluginMetadata.Disabled)
         {
-            if (context.CurrentPluginMetadata.Disabled)
-            {
-                // Don't load or monitor files if disabled
-                return;
-            }
-
-            cachedBookmarks = BookmarkLoader.LoadAllBookmarks(_settings);
-            _ = MonitorRefreshQueueAsync();
-            initialized = true;
+            // Don't load or monitor files if disabled
+            return;
         }
 
-        public List<Result> Query(Query query)
+        _cachedBookmarks = BookmarkLoader.LoadAllBookmarks(_settings);
+        _ = MonitorRefreshQueueAsync();
+        _initialized = true;
+    }
+
+    public List<Result> Query(Query query)
+    {
+        // For when the plugin being previously disabled and is now reenabled
```

Comment:
maybe a hyphen between re and enabled? I don't feel like this is a word...correct me if i am wrong

Reply:
You're correct. Added a hyphen.